aiida_common_workflows/cli/launch.py

```diff
@@ -39,6 +44,26 @@ def cmd_relax(plugin, structure, protocol, relaxation_type, threshold_forces, th
     process_class = load_workflow_entry_point('relax', plugin)
     generator = process_class.get_inputs_generator()
 
+    number_engines = len(generator.get_calc_types())
+
+    if number_machines is None:
+        number_machines = [1] * number_engines
+
+    if len(number_machines) != number_engines:
+        raise click.BadParameter(
+            f'{process_class.__name__} has {number_engines} engine steps, so requires {number_engines} values',
```

Comment:
I would just specify that a list is needed.
Something like:
`'{process_class.__name__} has {number_engines} engine steps, so requires {number_engines} values in a list'

Reply:
but that might be confusing no? They might think they have to pass `--number-machines=[1, 2]` which is incorrect. It should just be `--number-machine 1 2`. At least that is what I would read if I saw `list`

Reply:
My bad, I didn't read properly the command features. Forget about it